client/src/Views/ReportsPage/AllShouts.vue

```diff
@@ -57,7 +63,7 @@ export default {
       return this.$store.state.loading.shoutouts;
     },
     shoutouts() {
-      return this.$store.state.shoutouts;
+      return this.$store.state.shoutouts.map(({ date, shoutee, shouter, shoutout, value }) => ({ date, shoutee, shouter, shoutout, value }));
```

Comment:
At some point we should update this so that it is formatted before it gets put into state